### PR TITLE
Objectives for Java lessons 1-3

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[Unit 2 work plan and learning objectives](https://trello.com/c/SR4W73Ui/196-unit-2-work-plan-and-learning-objectives)

File: unit2-java/lesson04.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Create constant fields using ``static final``
+- Explain why instance variables may not be referenced within static methods
+- Write a custom ``toString`` method for a given class
+- Explain the concepts of object identity and equality, and how they differ
+- Explain and properly use ``.equals`` to determine when two objects are equal
+- Explain the behavior of ``==`` with respect to objects in Java
+- Describe and implement the best practices for implementing ``.equals`` within a class
+- Explain when ``.hashCode`` should be implemented within a class
+- Use IntelliJ’s code generation tools to generate well-structured ``.equals`` and ``.hashCode`` methods
+- Use the Single Responsibility Principle to create small, well-designed classes
```
Comment (CarlyLanglois) [inline unit2-java/lesson04.rst]: looks great! my only question would be is this more fitting for lesson 3? or are there concepts introduced here in 4 that warrant pushing this into this lesson?

**Review by CarlyLanglois — APPROVED** [review]
lessons 1-4 are reviewed - just one question

File: unit2-java/lesson04.rst (hunk 1 above)
Comment (chrisbaylewis) [reply unit2-java/lesson04.rst]: Good call. I'll move SRP up into lesson 3.

**Pull request merged**